tests/test_graphql_api/test_reservation/test_query.py

```diff
@@ -138,7 +138,7 @@ def test_reservation__query__all_fields(graphql):
         "reserveeId": reservation.reservee_id,
         "reserveeIsUnregisteredAssociation": reservation.reservee_is_unregistered_association,
         "reserveeLastName": reservation.reservee_last_name,
-        "reserveeName": f"{reservation.reservee_first_name} {reservation.reservee_last_name}",
+        "reserveeName": f"{reservation.user.first_name} {reservation.user.last_name}",
```

Comment:
`reservee_first_name` + `reservee_last_name` is not the same as `user.first_name` + `user.last_name` . The former names are saved to the reservation at the time of creation, which they might be changed after. @vergama can also check if this should be the case here.

Reply:
This was an intentional change, as the `reservee_type` here is `None`, so user's name is used as fallback when we don't know if the reservee is an individual or organisation (and also `reservation.name` is not defined)